src/netius/test/common/http2.py

```
@@ -484,6 +714,355 @@ def test_parse_window_update(self):
+            # the end of stream completes the request and so the message is
+            # handed over to the upper layers as a complete one
+            stream = parser.streams[1]
+            self.assertEqual(events, ["data"])
+            self.assertEqual(stream.method, "POST")
+            self.assertEqual(stream.content_l, 11)
+            self.assertEqual(stream.get_message_b().read(), b"Hello World")
+        finally:
+            parser.clear(force=True)
+
+    def test_parse_headers(self):
+        if hpack == None:
```

Ruff (0.16.2)

[error] Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)